lib/Controller/ClientEditController.php

```diff
@@ -91,7 +94,9 @@ public function __invoke(ServerRequest $request)
                 $data['scopes'],
                 (bool) $data['is_enabled'],
                 (bool) $data['is_confidential'],
-                $data['auth_source']
+                $data['auth_source'],
+                //TODO: can this be edited, and if so by the owner or just the ssp admin?
+                $data['owner'] = $this->authContextService->isSspAdmin() ? $data['owner'] : $client->getOwner()
```

Comment:
IMHO owner is not a mutable field. If a RP has a new admin, he/she can register a new client and avoid to bother admins.

Reply:
I've removed the ability to edit owner.